packages/devops/configs/servers.conf

```diff
@@ -1,25 +1,14 @@
-server {
-  listen 80 default_server;
-  listen [::]:80 default_server;
-  return 301 https://$host$request_uri;
-}
 server {
 
-  listen       443 ssl http2;
-  listen       [::]:443 ssl http2;
+  listen       80;
```

Comment:
Removing http2 rationale:
- inter-service communication does not need to be http2
- http2 requires ssl (most browsers enforce this, but our node client may allow it)
- https://serverfault.com/a/765337